en/guide/routing.md

```diff
@@ -115,7 +115,7 @@ Nuxt.js lets you create nested route by using the children routes of vue-router.
 
 To define the parent component of a nested route, you need to create a Vue file with the **same name as the directory** which contain your children views.
 
-<p class="Alert Alert--info">Don't forget to write `<nuxt-child/>` inside the parent component (.vue file).</p>
+<p class="Alert Alert--nuxt-green"><b>Info:</b> don't forget to write `<nuxt-child/>` inside the parent component (<code>.vue</code> file).</p>
```

Comment:
Because it's an info, maybe we can use the « Info » design and not use the « Warning » design (see above).